src/Management/Application/PublishManagedModel.php

```diff
@@ -0,0 +1,21 @@
+<?php
+
+namespace Thinktomorrow\Chief\Management\Application;
+
+use Thinktomorrow\Chief\Audit\Audit;
+use Thinktomorrow\Chief\States\PageState;
+use Thinktomorrow\Chief\Management\ManagedModel;
+use Thinktomorrow\Chief\Management\Events\ManagedModelArchived;
+
+class PublishManagedModel
+{
+    public function handle(ManagedModel $model)
+    {
+        (new PageState($model))->apply('publish');
+        $model->save();
```

Comment:
Bij DeleteManagedModel checken we of het model een instance is van Statefullcontract. Is dat hier ook niet nodig?
Hetzelfde voor UnarchiveManagedModel?